config-templates/module_oidc.php

```diff
@@ -43,6 +43,12 @@
 
     // useridattr is the attribute-name that contains the userid as returned from idp
     'useridattr' => 'uid',
+    'permissions' => [
```

Comment:
I would add a short explanation about this config option, like 'option to allow users to register their own clients...', and also short indication on how to gracefully disable this feature.

Reply:
Make sense.